kie-wb-tests/pom.xml

```diff
@@ -52,6 +53,60 @@
         <artifactId>kie-wb-tests-gui</artifactId>
         <version>${version.org.kie}</version>
       </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-drools-wb-webapp</artifactId>
+        <version>${deployable.version}</version>
+        <classifier>${deployable.classifier.swarm}</classifier>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-wb-webapp</artifactId>
+        <version>${deployable.version}</version>
+        <classifier>${deployable.classifier.swarm}</classifier>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-wb</artifactId>
+        <version>${deployable.version}</version>
+        <classifier>${deployable.classifier.wildfly10}</classifier>
+        <type>war</type>
+      </dependency>
```

Comment:
I am probably missing something here. @tlivora could you please clarify why we need these? The versions of `kie-wb` WARs should already be in the kie-parent.